src/views/Feed/Feed.js

```diff
@@ -0,0 +1,56 @@
+import _ from 'lodash'
+import cx from 'classnames'
+import React, { PropTypes } from 'react'
+
+import META from '../../utils/Meta'
+import { getUnhandledProps } from '../../utils/propUtils'
+import * as sui from '../../utils/semanticUtils'
+import FeedContent from './FeedContent'
+import FeedDate from './FeedDate'
+import FeedEvent from './FeedEvent'
+import FeedExtra from './FeedExtra'
+import FeedLabel from './FeedLabel'
+import FeedLike from './FeedLike'
+import FeedMeta from './FeedMeta'
+import FeedSummary from './FeedSummary'
+import FeedUser from './FeedUser'
+
+function Feed(props) {
+  const { children, className, size } = props
+  const classes = cx('ui', className, size, 'feed')
+  const rest = getUnhandledProps(Feed, props)
+
+  return <div {...rest} className={classes}>{children}</div>
+}
+
+Feed._meta = {
+  library: META.library.semanticUI,
```

Comment:
Heads up, I've deprecated library as of today in #380.  See that PR for rationale.  There is also a failing test for including it now.